src/pages/trip/log-list.tsx

```diff
@@ -2,18 +2,28 @@ import React from 'react';
 import styled from '@emotion/styled';
 import Graph from 'components/graph';
 import { GiveNTakeAmountProps } from 'components/graph';
+import { Animals } from 'components/profile';
 import Log from './log';
 
-const DUMMY = [
-  { id: 1, expender: '지형', profile: 'puppy', amount: 20000, desc: '휴게소', date: '2021-04-16' },
-  { id: 2, expender: '주예', profile: 'hamster', amount: 8000, desc: '아메리카노', date: '2021-04-16' },
-  { id: 3, expender: '유진', profile: 'rabbit', amount: 180000, desc: '게스트하우스', date: '2021-04-16' },
-  { id: 4, expender: '유진', profile: 'rabbit', amount: 180000, desc: '게스트하우스', date: '2021-04-15' },
-  { id: 5, expender: '유진', profile: 'rabbit', amount: 180000, desc: '게스트하우스', date: '2021-04-15' },
-  { id: 6, expender: '유진', profile: 'unicorn', amount: 180000, desc: '게스트하우스', date: '2021-04-15' },
-  { id: 7, expender: '유진', profile: 'rabbit', amount: 180000, desc: '게스트하우스', date: '2021-04-15' },
-  { id: 8, expender: '유진', profile: 'bear', amount: 180000, desc: '게스트하우스', date: '2021-04-15' },
-  { id: 9, expender: '유진', profile: 'rabbit', amount: 180000, desc: '게스트하우스', date: '2021-04-15' }
+interface Dummy {
+  id: number;
+  expender: string;
+  profile: keyof typeof Animals;
+  amount: number;
+  desc: string;
+  date: string;
+}
+
+const DUMMY: Dummy[] = [
+  { id: 1, expender: '지형', profile: 'Puppy', amount: 20000, desc: '휴게소', date: '2021-04-16' },
```

Comment:
```suggestion
  { id: 1, expender: '지형', profile: Animals.Puppy, amount: 20000, desc: '휴게소', date: '2021-04-16' },
```
이것도 요렇게 바꾸면 될 것 같아유!

Reply:
옷 근데 저희 백엔드에서 동물 이름 string을 받기로 했는데, enum 형식 그대로 받는거도 가능한건가용??

Reply:
결국 `enum`은 평가될 때, value에 있는 값들로 평가되기 때문에 그냥 `string` 으로 넣는 것과 별 차이 없을 것 같습니다!

Reply:
오 그렇군요!! 괜히 고민했네요 ㅎㅎ 수정해두겠습니당!!

Reply:
@evan-moon f38af8a 요 커밋에 반영되었습니당!